Update to step-by-step instructions

fischermario opened the pull request:
Hi,

I have tried your instructions on a Jetson Nano with a NCS2 attached (please do not ask why). During the configuration I hit some road blocks and I saw the opportunity to fix some stuff in your instructions.

Please comment.

Best regards,
Mario

skhameneh commented:
Hey Mario,

Awesome!  Thanks for the PR, can you verify the Jetson Nano lacks NEON support?
The ARM A57 on board _likely_ has NEON support.

Thanks,

Shawn

fischermario commented:
Hey Shawn,

that is a very good question. I am sure that the Tegra X1 SoC has NEON support, but the OpenCV cmake configuration could not detect NEON which caused a critical failure. My solution was to simply disable NEON. This strange behavior may be caused by the schroot environment, but I do not have enough experience to really investigate. Sorry.

Best regards,
Mario

fischermario commented:
I have updated the instructions in respect to the new version OpenVINO 2019 R1 and I have corrected some minor mistakes.

Would you consider merging these changes?

ffaucouneau commented:
Hey!

Agree with  fischermario . Could be a good idea to merge the change?

skhameneh commented:
I'll take a look, thanks for the contributions!

fischermario commented:
I have updated the instructions to resemble the current version OpenVINO 2019 R2. I have switched the schroot environment to Debian Buster because it ships with Python 3.7 (no repo hacks necessary) and it allows OpenCV to compile successfully with NEON.

skhameneh commented:
Thanks @fischermario
